src/icrar/leap-accelerate/math/linear_math_helper.h

```diff
@@ -38,9 +38,19 @@ namespace icrar
     Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> ConvertMatrix(const casacore::Matrix<T>& value)
     {
         auto shape = value.shape();
-        auto output = Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic>(shape[0], shape[1]);
-        std::copy(value.begin(), value.end(), output.reshaped().begin());
-        return output;
+        auto m = Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic>(shape[0], shape[1]);
+
+        auto it = value.begin();
+        for(int col = 0; col < shape[1]; ++col)
+        {
+            for(int row = 0; row < shape[0]; ++row)
+            {
+                //eigen column major format
+                m(row, col) = *it;
+                it++;
+            }
+        }
+        return m;
```

Comment:
Can you remind me again what's the story with column/row major? If both casacore and eigen are column major I'd assume you can keep using `std::copy`.

Reply:
it's just copied code I wrote a while back before that pull request, will update